content/en/getting_started/synthetics/browser_test.md

```diff
@@ -103,21 +100,17 @@ The example below demonstrates how to map a user journey from adding an item to
 
 It is important to finish your browser test with an **Assertion** to ensure your application resulted in the expected state after the defined user journey.
 
-{{< img src="getting_started/synthetics/record-test.gif" alt="Record test steps"  style="width:90%;">}}
+{{< img src="getting_started/synthetics/record-test.mp4" alt="Record test steps"  video="true"  >}}
 
 **Note**: The example website regularly throws an error causing it to intentionally fail. If you include your email address in the **Notify your team** field, you will receive an email notification when the test fails and recovers.
 
 ## Test results
 
 The Browser Test details page displays an overview of your test configuration, the global and per location uptime, graphs about time-to-interactive and test duration, sample successful and failed test results, and the list of all test results. Depending on the length of your test, you might have to wait for a few minutes to see the first test results come in.
 
-To troubleshoot a [failed test][6], select a failed test result and start by reviewing screenshots, including the ones leading up to the failed step. You can also review potential **[Errors & Warnings][7]**, **[Resources][8]** as well as **[Core Web Vitals][9]** to diagnose the issue. With Datadog's [APM integration with Synthetic Monitoring][10], you also have the ability to look into the traces that are generated in your backend by the test runs from the **Traces** tab.
-
-In the example below, the test failed as the result of a server timeout.
-
-{{< img src="getting_started/synthetics/browser-test-failure.png" alt="Browser test failure"  style="width:100%;">}}
```

Comment:
I'd be in favor of keeping a screenshot/mp4 to show test results. If you're looking for a good example, tjd-upf-kvk on the demo account is a great one! Let me know if you need help figuring out what we should show in there (I think going over the screenshots, core web vitals, errors, resources, and traces should be sufficient!)

Reply:
Hey @MargotLepizzera, agreed that a screenshot here would be helpful but the UI has changed; when you click on a Test Result, the Test Details panel no longer appears. Instead, the Step Results page loads and displays errors, resources, and traces. 

Ideally, I'd like a screenshot where the step itself is failing and there are errors/warnings to display. In the demo test above, I see Step 11 failing, but there are no errors, resources, or traces for me to click or preview. Can you provide a screenshot or video for a real failed browser test step?

Reply:
I'm not sure to understand the bit about the UI change. But what you can do is indeed pick a failing test result, and show errors/resources/traces. It's expected that in general errors/resources/traces show up at the step before the one that failed (a resource is not loading properly, so the following step is failing). Something along those line should work: https://a.cl.ly/geugDRpb. For reference, I used test m9g-88u-k7h on the demo account.